auth/src/lib.rs

```diff
@@ -298,7 +299,8 @@ where
 
         let mut response = Response::new(UserResponse {
             account_name: name.to_string(),
-            key: key.to_string(),
+            // TODO: change this to .expose() when #925 is merged.
```

Comment:
Yes, we can do that! I think we will also be forced to address it when that PR is merged since it will fail to compile. I think I will leave a (different) comment there to make it clear that it has to be `as_ref` currently, or else it will return a REDACTED string as the key in the response.